src/control/Control.Layers.js

```diff
@@ -204,7 +204,15 @@ export var Layers = Control.extend({
 		link.setAttribute('role', 'button');
 
 		DomEvent.on(link, 'click', DomEvent.preventDefault); // prevent link function
-		DomEvent.on(link, 'focus', this.expand, this);
+		DomEvent.on(link, 'keydown', function (e) {
+			if (e.keyCode === 13) {
+				DomEvent.on(section, 'click', DomEvent.preventDefault);
+				this.expand();
+				setTimeout(function () {
+					DomEvent.off(section, 'click', DomEvent.preventDefault);
+				});
+			}
+		}, this);

```

Comment:
Thanks @larsgw, however this doesn't work using a screen reader, see related  https://github.com/Leaflet/Leaflet/pull/8202#pullrequestreview-963070739.

Reply:
I see. I don't have Windows so I don't think I can test NVDA or Narrator, but I'll try to reproduce with Orca and take a look.